testproject/Assets/Tests/Runtime/NetworkSceneManager/InScenePlacedNetworkObjectTests.cs

```diff
@@ -18,22 +18,27 @@ namespace TestProject.RuntimeTests
     [TestFixture(NetworkTopologyTypes.ClientServer, HostOrServer.Server)]
     public class InScenePlacedNetworkObjectTests : IntegrationTestWithApproximation
     {
-        protected override int NumberOfClients => 3;
+        protected override int NumberOfClients => 2;
```

Comment:
Just curious why you reduced the client count here?

Reply:
This test was creating three clients but set `m_CanStartServerAndClients` to block automatically starting anything. The test then saved the third client and removed it manually from `m_ClientNetworkManagers` before calling `StartServerAndClients()`. It would later start that saved third client as a late join client.

Introducing `m_NetworkManagers` broke this flow as `StartServerAndClients()` was failing because that third client (which was still tracked in `m_NetworkManagers`)  hadn't been started, only the clients in the manually updated `m_ClientNetworkManagers` were started.

I updated the test to only use two clients and then to use `CreateAndStartNewClient()` to handle late joining a client.

Reply:
Ahhh... I see... yeah... makes sense!
I most likely wrote that when `CreateAndStartNewClient` didn't do all of the fancy stuff it does now... 👍